ui/accordion-collapsible-panel.reel/accordion-collapsible-panel.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @module ui/collapsible-panel.reel
+ * @requires montage/ui/component
+ */
+var Component = require("../component").Component;
+
+/**
+ * @class AccordionCollapsiblePanel
+ * @extends Component
+ *
+ * The internal panel inside Accordion to display header abd body
+ */
+exports.AccordionCollapsiblePanel = Component.specialize(/** @lends AccordionCollapsiblePanel# */ {
+    constructor: {
+        value: function CollapsiblePanel() {
+            this.super();
+        }
+    },
+
+    /**
+     * The parent Accordion component contains this.
+     * @type {Accordion}
+     */
+    accordion: {
+        value: null
+    },
+
+    /**
+     * Is the collapsible panel expanded or collapsed
+     * @type {boolean}
+     */
+    isExpanded: {
+        set: function (value) {
+            this._isExpanded = value;
```

Comment:
Can the header access the isExpanded property? They may want to bind it to a class so they can style expanded and contracted headers differently.